db/migrations/9.4.04.sql

```diff
@@ -0,0 +1,247 @@
+DO $$
+BEGIN
+   IF EXISTS (
+      SELECT 1
+      FROM information_schema.columns 
+      WHERE table_schema = 'public'
+            AND table_name = 'accounts'
+            AND column_name = 'identity_id'
+      ) 
+   THEN 
+      IF EXISTS (
+         SELECT 1
+         FROM information_schema.columns 
+         WHERE table_schema = 'public'
+               AND table_name = 'accounts'
+               AND column_name = 'identity_id'
+               AND is_nullable = 'NO' )
+      THEN
+         ALTER TABLE public.accounts ALTER COLUMN identity_id DROP NOT NULL;
+      END IF;
+   END IF;
+END $$;
+
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE IF NOT EXISTS account_history (
+    id text NOT NULL DEFAULT uuid_generate_v4(),
+    account_id text,
+    identity_id text,
+    event_id public_enum_8f5a39c8ee,
+    permissions_id text,
+    datetime timestamp(6),
+    updated_block_id text
+);
+
+BEGIN;
+
+-- handle primary key updated
+
+-- create account history entries
+WITH event_data AS (
+  SELECT
+    event_arg_0 as did,
+    event_id,
+    block_id as updated_block_id,
+    created_at as updated_at
+  FROM events
+  WHERE event_id = 'PrimaryKeyUpdated'
+  ORDER BY created_at ASC
+), account_data AS (
+  SELECT
+    accounts.id,
+    accounts.permissions_id
+  FROM accounts
+  JOIN event_data ON accounts.identity_id = event_data.did
+)
+INSERT INTO account_history (account_id, identity_id, event_id, permissions_id, datetime, updated_block_id)
```

Comment:
By default SQ adds `created_at` and `updated_at` column to each table in SQ, those details are missing in the create table query as well and in the insert query as well